phpcs.xml

```diff
@@ -1,27 +1,36 @@
 <?xml version="1.0"?>
-<ruleset name="Coding standards for the WP GraphQL plugin">
-
+<ruleset name="Coding standards for WPGraphQL">
```

Comment:
Quick tip, you can use `<arg name="parallel" value="20"/>` within the ruleset to have the lints run in parallel, which makes them run much faster! I just found out about this the other day, so I just figured I'd pass it along! 😄